src/Daml.Ledger.Api.Data/DamlCollectors.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright(c) 2019 Digital Asset(Switzerland) GmbH and/or its affiliates.All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace Daml.Ledger.Api.Data
+{
+    public sealed class DamlCollectors

```

Comment:
Can you remove this file then for now?

Reply:
It would be good to leave it as it is in the matching 0.13.41 java repo, makes it easier to keep the C# repo in step with the java repo.